packages/itwin/quantity-formatting/src/components/quantityformat/FormatSetPanel.tsx

```diff
@@ -0,0 +1,164 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from "react";
+import { SvgHelpCircularHollow } from "@itwin/itwinui-icons-react";
+import { ComboBox, Flex, IconButton, Input, Label, Text, Textarea } from "@itwin/itwinui-react";
+import { useTranslation } from "../../useTranslation.js";
+
+import type { UnitSystemKey } from "@itwin/core-quantity";
+import type { FormatSet } from "@itwin/ecschema-metadata";
+/**
+ * Props for the FormatSetPanel component
+ * @internal
+ */
+interface FormatSetPanelProps {
```

Comment:
I think props could be imroved to require `onFormatSetChange` when `editable` is `true`:
```ts
type FormatSetPanelProps = {
  formatSet: FormatSet;
} & ({ 
  editable: true,
  onFormatSetChange: (formatSet: FormatSet) => void 
} | {
  editable?: false,
  onFormatSetChange?: undefined 
})
```